dll/win32/kernel32/kernel32.spec

```diff
@@ -1193,10 +1193,10 @@
 @ stdcall WinExec(str long)
 @ stdcall Wow64DisableWow64FsRedirection(ptr)
```

Comment:
All those Wow64 functions quite probably need an extra `-arch=x86_64`.
Cc @tkreuzer for confirmation.